pages/quick-start/android-firebase-credentials.md

```diff
@@ -21,16 +21,16 @@ The credential file can be generated from the Firebase [console](https://console
 
 - Enter your application details then click on 'Register app':
 
-> The debug signing certificate is optional to use Firebase with your app, but is required for Dynamic Links, Invites and Phone Auth. To generate a certificate run `cd android && ./gradlew signingReport` and copy the SHA1 from the `debug` key.  
+> The debug signing certificate is optional to use Firebase with your app, but is required for Dynamic Links, Invites and Phone Auth. To generate a certificate run `cd android && ./gradlew signingReport` and copy the SHA1 from the `debug` key.  This generate two variant keys. You can copy the 'SHA1' that belong to the `debugAndroidTest` variant key option
 
 ![Register app](https://prismic-io.s3.amazonaws.com/invertase%2F3ea8d102-0fa8-4a5e-bbb4-938f5955800e_screenshot+2019-05-07+at+11.00.01.png)
 
-- Download the config file and save it to the `android/<myProjectName>/` folder in your project:
+- Download the google-services.json config file for firebase and save it to the `<myProjectName>/android/app` folder in your project:
 > Substitute `<myProjectName>` for the name of your new project.
 
 ![Download](https://prismic-io.s3.amazonaws.com/invertase%2F3bd36734-a2a7-46c7-aca5-a7b007c4ec35_screenshot+2019-05-07+at+10.48.12.png)
 
-- Ignore the next steps on the Firebase Console and load the `google-services.json` file with the Google services plugin for Gradle:
+- From the next step, get the com.google.gms:google-services version number, to replace the undefined in the image above. Ignore the rest of the details in this next steps on the Firebase Console and load the `google-services.json` file with the Google services plugin for Gradle:
```

Comment:
I don't understand this one - get the version number from where? where is the undefined in the image? (I looked and couldn't see one?)

Reply:
<img width="1307" alt="Screenshot 2019-08-31 at 4 13 23 PM" src="https://user-images.githubusercontent.com/6086663/64065819-78352980-cc0a-11e9-9429-2e916c0f56d0.png">
In the documentation, it does not specify what google.gms:google-services version number to use, instead, it shows undefined as shown in the attached image.  To make the documentation work for me, i had  to get the version code from firebase developer console step i was advice to ignore in the documentation.
<img width="1343" alt="Screenshot 2019-08-31 at 4 21 47 PM" src="https://user-images.githubusercontent.com/6086663/64065906-8e8fb500-cc0b-11e9-9886-e7ac95552194.png">

If i don't ignore this step, i was able to see com.google.gms:google-services version number on firebase console.

Reply:
Oh interesting! @Ehesp this is probably a problem with version numbers interpolating from some version YAML into the docs? And should be fixed via fixing the version number interpolation vs adding docs?

Reply:
I think something like here https://github.com/react-native-community/react-native-localize/pull/65 - on the v5 docs it would be like so: https://github.com/invertase/react-native-firebase-docs/blob/v5.x.x/docs/_config.yaml#L24 - but maybe the v6 docs aren't fully configured there yet

Reply:
Yep looks like it's not pulling through, I'll get it checked out.